src/server/ServerMethod.js

```diff
@@ -87,6 +87,7 @@ export default class ServerMethod {
             path: url,
             params: urlParams
           },
+          postBody: args,
```

Comment:
I was going with that originally, but it's not really an argument - its the entire post body in this case. I'm happy with either one tho.

I'm still totally not sure why this isn't being passed into the method in my code in the first place...